app/src/main/java/com/example/infinite_track/presentation/screen/wfa/components/WfaRequestCard.kt

```
@@ -69,185 +79,167 @@ fun WfaRequestCard(
 private fun ScoreBadge(score: Double?, color: Color) {
     Box(
         modifier = Modifier
-            .size(42.dp)
+            .size(30.dp)
             .clip(CircleShape)
-            .background(color.copy(alpha = 0.11f)),
+            .border(BorderStroke(1.5.dp, color), CircleShape),
         contentAlignment = Alignment.Center
     ) {
         Text(
             text = score?.let { String.format("%.1f", it) } ?: "-",
```

detekt (1.23.8)

[warning] String.format("%.1f", it) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)